runtime/stdlib/webassembly.cdc

```diff
@@ -0,0 +1,23 @@
+access(all)
+contract WebAssembly {
+
+    /// Compile WebAssembly binary code into a Module
+    /// and instantiate it with the given imports.
```

Comment:
Do we (plan to) support imports here? I thought there won't be any imports. At least for the first cut.

Reply:
Ah, good catch. I had started writing out the API here, and had initially included support for e.g. imports, but then revised the API to only what is supported by the PoC, but forgot to update the comments. 
```suggestion
    /// and instantiate it. Imports are not yet supported.
```